<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchUserDialog</class>
 <widget class="QDialog" name="BatchUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批量添加用户</string>
  </property>
  <widget class="QPushButton" name="ptnDelete">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnModify">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnAdd">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnSave">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnClose">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="userTableWidget">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>130</y>
     <width>620</width>
     <height>470</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>用户编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>用户名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>备注</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>密码</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>31</y>
     <width>151</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户编号*：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtUserNo"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="text">
       <string>由0-9的数字组成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>用户密码*：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtUserPwd">
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>527</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>备注：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtUserComments"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>用户名称*：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtUserName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
